tests/expectations/metrics/test_metrics_util.py

```diff
@@ -272,3 +295,149 @@ def test_get_unexpected_indices_for_multiple_pandas_named_indices_named_unexpect
         "Error: The list of domain columns is currently empty. Please check your "
         "configuration."
     )
+
+
+@pytest.mark.unit
+def test_get_dbms_compatible_metric_domain_column_kwargs_column_not_found(
+    sa, column_names_all_lowercase: list[str]
+):
+    test_column_names: list[str] = column_names_all_lowercase
+    with pytest.raises(gx_exceptions.InvalidMetricAccessorDomainKwargsKeyError) as eee:
+        _ = get_dbms_compatible_metric_domain_kwargs(
+            metric_domain_kwargs={"column": "non_existent_column"},
+            batch_columns_list=test_column_names,
+        )
+    assert (
+        str(eee.value)
+        == f'Error: The column "non_existent_column" in BatchData does not exist.'
+    )
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "input_column_name, output_column_name, confirm_not_equal_column_name",
+    [
+        [
+            "ARTISTS",
+            "ARTISTS",
+            None,
+        ],
+        [
+            "travel_agents",
```

Comment:
I'm going to approve, but in the future let's use more descriptive names here. E.g. 
```suggestion
            "should_be_quoted",
```
It really helps with readability.

Reply:
@anthonyburdi I will make this change in a tiny pull request imminently.  Thanks!